examples/filters/pycolorfilter/requirements.txt

```diff
@@ -0,0 +1,10 @@
+googleapis-common-protos==1.60.0
+grpclib==0.4.5
+h2==4.1.0
+hpack==4.0.0
+hyperframe==6.0.1
+multidict==6.0.4
+Pillow==10.0.0
+protobuf==4.24.0
+typing_extensions==4.7.1
+/Users/katherinewu/viam/viam-python-sdk
```

Comment:
This is necessary to pull in [recent changes](https://viam.atlassian.net/browse/RSDK-4646) to the python SDK. Once these changes are included in a release this can be changed to `viam-python-sdk`